src/styles/main.scss

```diff
@@ -0,0 +1,82 @@
+$day_background-color: #eee;
+$day_block-size: 100px;
+$calendat-gap: 1px;
+$calendar-width: calc($day_block-size * 7 + $calendat-gap * 6);
+
+@mixin squere($size) {
+  height: $size;
+  width: $size;
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  box-sizing: border-box;
+  width: #{$calendar-width};
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} {
+      & .calendar__day:nth-child(n + #{$index + 1}) {
+        @extend %last-day;
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include squere($day_block-size);
+
+    background-color: #{$day_background-color};
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: all 0.5s;
+
+    @for $day from 1 through 31 {
+      &:nth-of-type(#{$day})::before {
+        content: '#{$day}';
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: #ffbfcb;
+    }
+  }
+
+  $days: 
+
```

Comment:
Unnecessary empty line. There is also eslint error in line 61 thats why github checks failed `(Expected single space after ":" in line 61)`